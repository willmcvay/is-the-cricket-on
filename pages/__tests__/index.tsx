import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import HomePage from '../index'

describe('HomePage', () => {
  const renderComponent = () => {
    return shallow(<HomePage />)
  }

  it('should match a snapshot', () => {
    const Component = renderComponent()
    expect(toJson(Component)).toMatchSnapshot()
  })
})
