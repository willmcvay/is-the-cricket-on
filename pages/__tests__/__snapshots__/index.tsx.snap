// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage should match a snapshot 1`] = `<Apollo(HomePageProvider) />`;
